libraries/usersearch/api/src/main/kotlin/io/element/android/libraries/usersearch/api/UserRepository.kt

```diff
@@ -16,10 +16,9 @@
 
 package io.element.android.libraries.usersearch.api
 
-import io.element.android.libraries.matrix.api.user.MatrixUser
 import kotlinx.coroutines.flow.Flow
 
 interface UserRepository {
```

Comment:
It's not new, but is `UserRepository` a find name? 
And do we really need to have a module just for that? Could we not embedded that inside the Matrix module?

Reply:
Florian suggested the name. It does some business logic to combine data from several sources, so seems reasonable?

It could go in the Matrix module, but it felt more like a "specific thing we do on top of Matrix in a small number of places" rather than a general purpose thing. If we pushed everything that had anything to do with Matrix into the Matrix module, 90% of the app could be in there :sweat_smile:

Reply:
(I'll go ahead and merge this to get it out the way. If we want to move it, it'll be easier in a separate PR with no changes to the logic)